### PR TITLE
Newsweek home page

### PR DESCRIPTION
**Pull request by helciodev**
In this project,  I was tasked to recreate a clone from the newsweek.com site homepage using front-end technologies like HTML and  CSS but mainly the framework from Twitter, Bootstrap. 
The project has the following features:

- Navbar with nav list items as blocks.
- Main section 
- footer with footer nav list items and footer nav links and social links.

### CODE REVIEW

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
### Hello @helciodev ! 

### You have done an astounding job in this project so far. However, there are some necessary changes to be made before it can be approved.

 
### Recommended




**Navigation Bar**
- [x] The info bar which holds the logo needs more padding and the logo itself should have a larger width.

- [x] The font-size of the "Search" placeholder in the nav-bar should be reduced.

- [x] The debate column in your project has a smaller width than the original.

**My Turn**
- [x] The images in the "My Turn" section are of different sizes, please make them uniform.

**More Stories**
- [x] The "More stories" section has a glitch, it's not gliding as smoothly as that of the actual page.

**Free Sign Up Form**
- [x] You have re-arranged the  position of the page content. The "Free Sign Up" Form should come after the "Opinions".

**The Magazine**

- [x] The images in this section should be larger. Refer to the [Newsweek site](https://www.newsweek.com/) to see the comparison.

**Newsletter**
- [x] This section needs more work as well. The image and form does not fit the proportions of the Newsweek site.

**Footer**

- [x] The footer also needs to be fixed. The font-size is larger than that of the actual site.

- [x] The logo has a different color from the actual site's.

**Screen Sizing**
- [x] The Debate section and the Opinions section in the medium screen size(768px ) should take the whole space.

- [x] You have extra features in your project that is not reflected in the Newsweek site, please remove those.

- [x] The Subscribe and Newsletter sections should fill the entire row. 

- [x] The footer is not displaying as it should.

 

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review @tamara._ :wink: 
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: assets/sass/style.css [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+  height: 90px;
+}
+
+.fixed--nav {
+  display: none;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  height: 45px;
+  background: #f72210 !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+  background-size: cover;
+}
+
+.logo-footer {
+  width: 17rem;
+  height: 3rem;
+}
+
+.social-footer .facebook:hover {
+  color: #3b5998 !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+}
+
+.footer-links > *:not(:last-child) {
+  margin-right: 20px;
+}
+
+.social-footer .twitter:hover {
+  color: #1da1f2 !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 4]
```diff
@@ -0,0 +1,563 @@
...
+}
+
+.bg-red {
+  background-color: red;
+}
+
+.social-footer .instagram:hover {
+  color: #405de6 !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 5]
```diff
@@ -0,0 +1,563 @@
...
+}
+
+.social-footer .tumblr:hover {
+  color: #3592d5 !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 6]
```diff
@@ -0,0 +1,563 @@
...
+}
+
+.social-footer .linkedin:hover {
+  color: #0077b5 !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 7]
```diff
@@ -0,0 +1,563 @@
...
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .navbar-side .nav-item:hover {
+    background-color: transparent;
+  }
+  .nav-bottom .nav-link:hover {
+    color: #f72210 !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 8]
```diff
@@ -0,0 +1,563 @@
...
+  .sticky {
+    position: fixed;
+    top: 0;
+    width: 448px;
+  }
+  .newsletter-element {
+    height: 172px;
+  }
+  .main {
+    margin-top: -4rem !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: assets/sass/style.css [hunk 9]
```diff
@@ -0,0 +1,563 @@
...
+  }
+  .text-abs-md {
+    font-size: 20px;
+    top: 70px;
+    width: 50% !important;
```
Comment (umairarshadbutt) [inline assets/sass/style.css]: Remove `!important`

File: index.html [hunk 10]
```diff
@@ -0,0 +1,2255 @@
...
+       
+
+        <ul class="list-unstyled d-flex">
+          <li class="text-dark font-weight-bold text-capitalize mr-3" >editions</li>
+          <li class="mr-3"><a class="text-secondary text-decoration-none" href="#">U.S. Edition</a></li>
+          
+          <li class="mr-3"><a class="text-secondary text-decoration-none" href="#">日本</a></li>
+          <li class="mr-3"><a class="text-secondary text-decoration-none" href="">Pakistan</a></li>
+          <li class="mr-3"><a class="text-secondary text-decoration-none" href="">Polska</a></li>
+          <li class="mr-3"><a class="text-secondary text-decoration-none" href="">
```
Comment (umairarshadbutt) [inline index.html]: Add `#` or `URL` in `href` attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @helciodev 

Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Update the `href` attribute as mentioned in inline comment.


### MAGAZINE Section
- [x] Add margin-top on other cards as highlighted in the screenshot

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://helciodev.github.io/newsweek-site-clone/)|
|:---:|:---:|
|source-page|clone|


### EDITOR'S PICK Section
- [x] Please add 4 cards in large screen as original page

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://helciodev.github.io/newsweek-site-clone/)|
|:---:|:---:|
|source-page|clone|



##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by ioilmio — APPROVED** [review]
## Congrats @helciodev you fix all the issues pointed out from the previous reviewers, your project is approved 👍🏼  🎉

**Pull request merged**